File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_UserPasswordHistoryService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_UserPasswordHistoryService : BaseService<Sys_UserPasswordHistory, ISys_UserPasswordHistoryRepository>
    , ISys_UserPasswordHistoryService
    {
    private readonly ISys_UserPasswordHistoryRepository _Sys_UserPasswordHistoryRepository;
    public Sys_UserPasswordHistoryService(ISys_UserPasswordHistoryRepository Sys_UserPasswordHistoryRepository) : base(Sys_UserPasswordHistoryRepository)
    {
    _Sys_UserPasswordHistoryRepository = Sys_UserPasswordHistoryRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_UserPasswordHistory.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_UserPasswordHistory } from './Sys_UserPasswordHistory.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_UserPasswordHistoryService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'User_Id', 'ChangedDate', 'ChangedDateTime', 'ExpirationDate', 'User_Password'];

    List: Sys_UserPasswordHistory[] | undefined;
    ListFilter: Sys_UserPasswordHistory[] | undefined;
    FormData!: Sys_UserPasswordHistory;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_UserPasswordHistory";
    }
}

